application/cms/forms.py

```diff
@@ -210,6 +211,18 @@ def __call__(self, form: "MeasureVersionForm", field):
         strip_whitespace=True,
     )
 
+    description = RDUTextAreaField(
+        label="Description for search engines",
+        validators=[RequiredForReviewValidator()],
+        hint=(
+            "Choose an up to date statistic that shows a key disparity or change over time. The figure should work as "
```

Comment:
change 'up to date' to 'up-to-date'

(we've checked, and it's probably best to hyphenate as it's a compound adjective before a noun)